### PR TITLE
Migrate CLX parsers

### PR DESCRIPTION
**Pull request by efajardo-nv**
## Description
- Splunk Notable parser
- Windows Event parser
- URL parser
- Zeek parser
- IP parser
- Unit tests

Tasks from Issue #508

## Checklist
[x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
[x] New or existing tests cover these changes.
[x] The documentation is up to date with these changes.

### CODE REVIEW

File: tests/test_splunk_notable_parser.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+             'category="riskware-callback", dest_ip="10.15.90.150", occurred="Mar 09 2019 02:36:00 UTC", ' \
+             'signature="Android.Adware.Batmobil", src_ip="10.15.90.151", dest_port="80", src_port="40472", ' \
+             'orig_time="1552098960", info_max_time="1552099380.000000000", info_min_time="1552098780.000000000", ' \
+             'info_search_time="1552052094.393543000", severity="minr", src_host="ip-10.5.13.compute.internal"'
+TEST_DATA7 = '1566345700, search_name=\\"Endpoint - Host With Malware Detected (Quarantined or Waived) - ' \
+             'Rule\\", count=\\"1\\", dest=\\"TEST-01\\", dest_priority=\\"medium\\", ' \
+             'info_max_time=\\"1511389440.000000000\\", info_min_time=\\"1511388840.000000000\\", ' \
+             'info_search_time=\\"1511389197.841039000\\", info_sid=\\"rt_scheduler_dGNhcnJvbGxAbnZpZGlhLmNvbQ__' \
+             'SplunkEnterpriseSecuritySuite__RMD5c5145919d43bdffc_at_1511389196_22323\\", ' \
+             'lastTime=\\"1511388996.202094\\"'
```
Comment (mdemoret-nv) [inline tests/test_splunk_notable_parser.py:73]: Should this be in a data file somewhere? Why is it inline?

**Review by mdemoret-nv — APPROVED** [review]
Looks good. My lone comment can be resolved if these tests came directly from CLX

**Comment by mdemoret-nv** [comment]
@efajardo-nv Does this close out #508? Or are there more to do?

**Comment by efajardo-nv** [comment]
> Does this close out #508? Or are there more to do?

No, couple more tasks to close out #508.

**Pull request merged**